helm/charts/hpe-csi-driver/values.yaml

```diff
@@ -33,4 +33,7 @@ kubeletRootDir: "/var/lib/kubelet/"
 disableNodeGetVolumeStats: false
 
 # CSP client timeout for HPE Alletra 9000, Primera and 3PAR (60-360 seconds)
```

Comment:
Can you remove the csp client timeout related section altogether , the comments as well

Reply:
removed